### PR TITLE
1336/tooltip date orders

### LINKED ISSUE #1336: Show tooltip for the date on orders
Same as with Trades

_Estimation: 0.5_

### PR DESCRIPTION
**Pull request by W3stside**
Closes #1336 

Adds locale aware date strings on `Status` and `Expire` cells of Trade page showing full date

**Not sure if this is what we wanted?** the issue says like in `Trades` page but that date is the date of the order being filled, aka block timestamp. Is this what we want?

Screenshot from 2020-10-07 20-19-59

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
   * **🎩 [Dapp](https://pr1507--dexreact.review.gnosisdev.com)**: Testing web app
   * **📚 [Storybook](https://pr1507--dexreact.review.gnosisdev.com/storybook/)**: Component stories

File: src/components/OrdersWidget/OrderRow.tsx [hunk 1]
```diff
@@ -151,14 +151,19 @@ const Amounts: React.FC<AmountsProps> = ({ sellToken, order }) => {
 }
 
 const Expires: React.FC<Pick<Props, 'order' | 'pending' | 'isPendingOrder'>> = ({ order, isPendingOrder }) => {
-  const { isNeverExpires, expiresOn } = useMemo(() => {
+  const { isNeverExpires, expiresOn, expireDateLocal } = useMemo(() => {
```
Comment (anxolin) [inline src/components/OrdersWidget/OrderRow.tsx:154]: nit: maybe `expireDateFormatted` instead of local?

File: config-default.yaml [hunk 2]
```diff
@@ -213,7 +213,8 @@ initialTokenList:
     addressByNetwork:
       '1': '0x6B175474E89094C44Da98b954EedeAC495271d0F'
       '4': '0x5592EC0cfb4dbc12D3aB100b257153436a1f0FEa'
-      '100': '0x44fA8E6f47987339850636F88629646662444217'
+      # Disabled DAI: It's more convenient to use wxDAI
```
Comment (anxolin) [inline config-default.yaml:216]: why this change is in this branch??

**Review by anxolin — APPROVED** [review]

**Review by Velenir — APPROVED** [review]

File: src/components/OrdersWidget/OrderRow.tsx (hunk 1 above)
Comment (W3stside) [reply src/components/OrdersWidget/OrderRow.tsx:154]: ok
